src/components/DisbursementInviteMessage/index.tsx

```diff
@@ -0,0 +1,156 @@
+import { Fragment, useEffect, useState } from "react";
+import { Card, RadioButton, Textarea } from "@stellar/design-system";
+import { useDispatch } from "react-redux";
+
+import { useRedux } from "hooks/useRedux";
+import { AppDispatch } from "store";
+import { getOrgInfoAction } from "store/ducks/organization";
+
+import "./styles.scss";
+
+interface DisbursementInviteMessageProps {
+  draftMessage?: string;
+  isEditMessage: boolean;
+  onChange?: (message: string) => void;
+}
+
+export const DisbursementInviteMessage = ({
+  draftMessage,
+  isEditMessage,
+  onChange,
+}: DisbursementInviteMessageProps) => {
+  console.log(draftMessage);
+  enum radioValue {
+    ORGANIZATION = "organization",
+    CUSTOM = "custom",
+  }
+
+  const standardOrgMessage =
+    "You have a payment waiting for you from the {{.OrganizationName}}. Click {{.RegistrationLink}} to register.";
+  const customMessagePlaceholder =
+    "Input your registration invitation message for this disbursement here in the appropriate language. Make sure to say which organization the transaction is coming from. Keep your message short. A link to the wallet will be appended at the end.";
+
+  const { organization } = useRedux("organization");
+  const dispatch: AppDispatch = useDispatch();
+
+  const [selectedOption, setSelectedOption] = useState(radioValue.ORGANIZATION);
+  const [customMessageInput, setCustomMessageInput] = useState(
+    customMessagePlaceholder,
+  );
+
+  useEffect(() => {
+    dispatch(getOrgInfoAction());
+  }, [dispatch]);
+
+  const handleOptionChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    if (event.target.value === radioValue.ORGANIZATION) {
+      setCustomMessageInput("");
+      setSelectedOption(radioValue.ORGANIZATION);
+    } else {
+      setSelectedOption(radioValue.CUSTOM);
+    }
+  };
+
+  const updateMessage = (updatedDisbursementInviteMessage: string) => {
+    // Updating parent
+    if (onChange) {
+      onChange(updatedDisbursementInviteMessage);
+    }
+  };
+
+  const updateCustomMessageInput = (
+    event: React.ChangeEvent<HTMLTextAreaElement>,
+  ) => {
+    const { value } = event.target;
+
+    setCustomMessageInput(value);
+    updateMessage(value);
+  };
+
+  return (
+    <>
+      <Card>
+        <div className="CardStack__card ReceiverInviteMessage">
+          <div className="CardStack__title">Customize invite</div>
+
+          <div className="Note">
+            You can use the organization default message or customized text to
+            invite your receivers to set up a wallet and receive funds for this
+            disbursement. It contains a link to the appropriate wallet at the
+            end of the message. Please check all values for accuracy.
+          </div>
+
+          {isEditMessage ? (
+            <>
+              <div className="ReceiverInviteMessage__options">
+                <RadioButton
+                  id="msg-std"
+                  name="receiver-message"
```

Comment:
Should we change the name to include disbursement? This is the same name we have for the `ReceiverInviteMessage` component options.